### ISSUE TITLE
Not Fetching Subtitles 

### ISSUE THREAD

**Issue by mslmn** [ReneR97/skillshare-downloader#7]
When I run the tool, it downloads all the videos but no subtitles are fetched.

**Comment by parvinders347** [comment]
why do you need subtitles if there is an option for language?

**Comment by mslmn** [comment]
Subtitles are essential for accessibility, and can be helpful in many other circumstances that I’m sure don’t need spelling out here. Besides, according to the README, the tool should be able to fetch subtitles when required, so not doing so counts as an issue doesn’t it?

**Comment by ReneR97** [comment]
It is an issue, they changed the way subtitles are handled and i couldnt yet find a good way to retrieve them

**Comment by Conkernel** [comment]
> It is an issue, they changed the way subtitles are handled and i couldnt yet find a good way to retrieve them

Hi. Any chance to fix the subtitles issue?

Thanks a lor for your work. I think it's one of the only tools that actually works for skillshare.>

Regards
